Assets/MixedRealityToolkit/Definitions/InputSystem/PointerBehavior.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Specifies how a pointer in MRTK's default input system behaves.
+    /// By default, pointer behaviors are set to Default.
```

Comment:
>/// By default, pointer behaviors are set to Default. [](start = 4, length = 53)

Super nit: I think this line doesn't add that much (I think it's expected that something that is named "default" is the default)

Reply:
I think maybe if the enum was called "NotTheDefaultJustKiddingItsActuallyTheDefault" then having this comment would be good (actually in general I think that it's a good callout, just not in this case because the naming is fairly obvious)

---
In reply to: [323509091](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5920#discussion_r323509091) [](ancestors = 323509091)